performance-tests/bench_2/common/json_conversion.h

```diff
@@ -23,18 +23,34 @@ bool json_2_idl(std::istream& is, IDL_Type& idl_value) {
 }
 
 template<typename IDL_Type>
-bool idl_2_json(const IDL_Type& idl_value, std::ostream& os, bool pretty = false) {
+bool idl_2_json(const IDL_Type& idl_value, std::ostream& os,
```

Comment:
You could make this a template on `Writer` vs `PrettyWriter` to avoid duplicating the code.